checks/http/roundcube.rb

```diff
@@ -0,0 +1,42 @@
+module Intrigue
+module Ident
+module Check
+class Roundcube < Intrigue::Ident::Check::Base
+
+    def generate_checks(url)
+    [
+        {
+            :type => "fingerprint",
+            :category => "application",
+            :tags => ["Email"],
+            :vendor => "Roundcube",
+            :product => "Roundcube",
+            :website => "https://roundcube.net/",
+            :match_details => "Roundcube title",
+            :version => nil,
+            :match_type => :content_body,
+            :match_content =>  /:: Welcome to Roundcube/i,
+            :paths => ["#{url}"],
+            :inference => false
+        },
+        {
+            :type => "fingerprint",
+            :category => "application",
+            :tags => ["Email"],
+            :vendor => "Roundcube",
+            :product => "Roundcube",
+            :website => "https://roundcube.net/",
+            :match_details => "Roundcube session cookie",
```

Comment:
good, this is helpful to the user https://monosnap.com/file/bVhMEcbsMOiH9A6pUN5EQcCzQgwNfo